industrial_robot_simulator/industrial_robot_simulator

```diff
@@ -34,7 +34,12 @@
 import rospy
 import copy
 import threading
-import Queue
+
+import sys
+if sys.version_info[0] == 2:
```

Comment:
Suggestion: use `sys.version_info.major` instead: intent is immediately clear and no magic nrs.